cmd/kube-apiserver/app/server.go

```diff
@@ -349,6 +353,7 @@ func CreateKubeAPIServerConfig(
 		}
 		issuer = serviceaccount.JWTTokenGenerator(s.Authentication.ServiceAccounts.Issuer, sk)
 		apiAudiences = s.Authentication.ServiceAccounts.APIAudiences
+		maxExpiration = s.Authentication.ServiceAccounts.MaxExpiration
```

Comment:
if non-zero, should verify this is:
* >= the default value (1 hour)
* <= the max value (2^32 seconds)

Reply:
lgtm otherwise